### ISSUE TITLE
PiGallery2 Locks up randomly opening big folder with files and throwing element parse errors

### ISSUE THREAD

**Issue by kardo22** [bpatrik/pigallery2#612]
**Describe the bug**
Using MYSQL as database, and turning on debug i see that sometimes when opening a folder with many pictures in the folder is randomly starts throwing a massive amount of 

"[xmldom error]  element parse error: Error: Hierarchy request error: Only one element can be added and only after doctype "

Like thousands of them. The Pigallery2 just keeps animating that it is loading but it can take up to a minute before it actually starts loading anything.

**Used app version**:
[App version: v1.9.5]

**Comment by kardo22** [comment]
Found this was more likely an unraid issue, i will open another ticket discussing a solution to make pigallery2 usable on unraid with massive amounts of files

**Comment by bpatrik** [comment]
Xmldom sounds to me 2 things:
Illfornatted gpx files (having a second thought the server might not actually open them, only the client)

Illfornatted exif/iptc data in the photo.
Or the app does not load enough bytes from the head of a photo, so the XML based metadata can't be parsed. There is a settings to increase the size of bytes to be read.

**Comment by MasticoreChen** [comment]
Hi
I encountered the same problem. and i using sqlite.
please tell me where is settings to increase the size of bytes to be read?
